state/dbstore.go

```diff
@@ -105,6 +107,17 @@ func (s *DBStore) Delete(key string) (err error) {
 	return s.db.Delete([]byte(key), nil)
 }
 
+// Iterate entries which has a specific prefix
+// The key and value may be modified in the iterFunc
+func (s *DBStore) Iterate(prefix string, iterFunc func([]byte, []byte)) (err error) {
```

Comment:
I think that @mortelli is working on state store Iterate function that will provide better control. Until that is merged, I think that this implementation can stay even it is not providing a way to return error from iterFunc and stop iteration gracefully.

Reply:
yeah i saw that PR. Most likely that PR will be merged first. So i will pull that inside.

Reply:
Great, thanks.